src/header/header.css

```diff
@@ -0,0 +1,101 @@
+header{
+    margin-top: 5rem;
+    height: calc(100vh - 5rem);
+    display: grid;
+    place-items: center;
+    background: var(--color-white);
+    transition: var(--transition);
+}
+
+.header-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+}
+
+.header-pf img{
+    width: 22rem;
+    height: 22rem;
+    object-fit: cover;
+}
+.header-pf{
+    border-radius: 50%;
+    overflow: hidden;
+    margin-bottom: 1.5rem;
+}
+
+
+.header-container p{
+    width: 35rem;
+    text-align: center;
+    margin: 0.6rem 0 2rem;
+}
+
+.header-cta{
+    display: flex;
+    align-items: center;
+    gap: 1.2rem;
+}
+
+.header-socials{
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    display: flex;
+    flex-direction: column;
+    gap:1rem;
+}
+
+.header-socials a{
+    width: 1.8rem;
+    aspect-ratio: 1/1;
+    background: var(--color-light);
+    color: var(--color-black);
+    display: grid;
```

Comment:
why here we should use grid??

Reply:
It is possible to achieve the same goal with the flexbox. but the grid is much more powerful than flex in some cases. 
In this case, it is better to continue with CSS-grid you will have other sections on this page which will make it easier.